### PR TITLE
🚀 Rock paper scissor

### LINKED ISSUE #677: Stone Paper Scissors with GUI
I do checked the repo and  found that there is [`stone paper scissors `](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Stone-Paper-Scissors-Game)  pygame included but its implemented using Command Line .
 > I will be using _pygame_ and _tkinter_ library to make same with **Graphical User Interface**
> It can be more interactive and playful

### PR DESCRIPTION
**Pull request by Lakhankumawat**
# Description
Added Stone Paper Scissor Game Using Tkinter 
Rock Paper Scissor For this i have made a folder named [`RockPaperScissor - GUI`](https://github.com/Lakhankumawat/Amazing-Python-Scripts/tree/RockPaperScissor/StonePaperScissor%20-%20GUI) and in it i have my RockPaperScissor.ipynb file alongwith README file . The code runs totally fine and shows the expected outputs.I have implemented it using Tkinter library .

✔️ Made an environment to play it easily
✔️ Single player game - Player VS Computer 
✔️ Choice for Computer is generated randomly using random lib
✔️ Scores Updated and displayed correctly after each turn 
## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/Lakhankumawat/Amazing-Python-Scripts/blob/RockPaperScissor/StonePaperScissor%20-%20GUI/README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: StonePaperScissor - GUI/RockPaperScissors .ipynb [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
```
Comment (kaustubhgupta) [inline StonePaperScissor - GUI/RockPaperScissors .ipynb:1]: .py file is needed and not notebooks

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
There are lot's of white space in your notebook apart from this it would be great if you add .py file instead any kind of notebook .

File: StonePaperScissor - GUI/RockPaperScissors .ipynb (hunk 1 above)
Comment (Kushal997-das) [reply StonePaperScissor - GUI/RockPaperScissors .ipynb:1]: > .py file is needed and not notebooks

@kaustubhgupta Exactly .py file is more comfort and i run this program in my local , i didn't find any errors except there are some useless white spaces are exits .

**Comment by Lakhankumawat** [comment]
> There are lot's of white space in your notebook apart from this it would be great if you add .py file instead any kind of notebook .

Okay @Kushal997-das  sir i m _removing notebook_ and adding **py** file instead .

**Comment by Lakhankumawat** [comment]
I think now it is okay 
- no useless indentations
- no useless white spaces( except where ever needed)

**Review by Kushal997-das — APPROVED** [review]
Looks good to me now 👍🏼 
@Lakhankumawat Wait for other mentors reviews .

**Review by kaustubhgupta — APPROVED** [review]
@Lakhankumawat Good job! 💯

**Comment by Lakhankumawat** [comment]
> @Lakhankumawat Good job! 💯

 > Thanks Sir 😌

**Review by santushtisharma10 — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**